docs/working-with-bee/upgrade.md

```diff
@@ -114,4 +58,13 @@ To upgrade your manual installation, simply stop Bee, replace the Bee binary and
 
 #### Docker
 
-To upgrade your docker installation, simply increment the version number in your configurations and restart.
\ No newline at end of file
+To upgrade your docker installation, simply increment the version number in your configurations and restart.
+
+### Upgrading from a mainnet v1.4.x series to a mainnet v1.5.x series
+
+Bee v1.5.0 contains a completely new data storage format called Sharky.
+
+As part of these changes, existing data must be migrated to the new data structure expected by
+the 1.5.x client. This will happen automatically, but **may require extra space** and cause a spike in cpu requirements for the duration of the migration.
+
+If you can not accomodate approximately 10x as much disk space as is currently being used by your Bee `datadir`, you may want to run `bee db nuke` before upgrading (but after stopping the Bee service) to resync your nodes content from the network. If you have **locally pinned content** please ensure you have a local backup so that you can restamp and restore it to the network in case of disaster.
```

Comment:
`10x` is not required. Sharky utilization is around 50%... so ideally `2x` should be enough... `3x` if you want to be safe

Reply:
cool, thanks dude